TravelAgencyService/src/test/java/cz/muni/fi/pa165/travelagency/travelagencyservice/ExcursionServiceTest.java

```diff
@@ -149,10 +154,54 @@ public void setup() {
     public void setupMocks() throws ServiceException {
         MockitoAnnotations.initMocks(this);
     }
+    
+    @Test
+    public void testCreateExcursion(){
+        excursionService.createExcursion(excursion1);
+        verify(excursionDao).create(excursion1);
+    }
+    
+    @Test
+    public void testRemoveExcursion(){
+        excursionService.createExcursion(excursion1);
+        
+        excursionService.removeExcursion(excursion1);
+        verify(excursionDao).remove(excursion1);
```

Comment:
skus otestovat aj to, ci sa ta exkurzia vymazala aj z nejakeho tripu, ktory ju obsahoval